celery/canvas.py

```diff
@@ -725,7 +910,7 @@ def prepare_steps(self, args, kwargs, tasks,
                 task = from_dict(task, app=app)
             if isinstance(task, group):
                 # when groups are nested, they are unrolled - all tasks within
-                # groups within groups should be called in parallel
+                # groups should be called in parallel
```

Comment:
Please revert this or describe why you think this comment is inaccurate.

Reply:
reverted